crates/wasm-pkg-loader/src/source/warg.rs

```diff
@@ -61,7 +62,13 @@ impl PackageSource for WargSource {
         Ok(info
             .state
             .releases()
-            .map(|release| release.version.clone())
+            .filter_map(|release| {
+                if release.yanked() {
+                    None
+                } else {
+                    Some(release.version.clone())
+                }
+            })
             .collect())
```

Comment:
I wonder if this should be configurable, e.g. an `include_yanked` options.

Reply:
Or maybe label yanked in the output list?

Reply:
Yeah but it also might make sense to allow (attempting to) fetching the actual artifact too.

Reply:
Pushed an update will list all versions and mark yanked ones.